src/bidmanager.js

```diff
@@ -46,33 +46,18 @@ function getBidders(bid) {
 function bidsBackAdUnit(adUnitCode) {
   const requested = $$PREBID_GLOBAL$$._bidsRequested
     .map(request => request.bids
-      .filter(adUnitsFilter.bind(this, $$PREBID_GLOBAL$$._adUnitCodes))
```

Comment:
This is the `adUnitsFilter` function:
```
export function adUnitsFilter(filter, bid) {
  return filter.includes((bid && bid.placementCode) || (bid && bid.adUnitCode));
}
```
so while it does behave somewhat differently by testing for `placementCode` and `adUnitCode` your right that the next filter cuts that down to just `placementCode`. Ok, makes sense.